app/main.py

```diff
@@ -1,8 +1,24 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = []
+    for i in people:
+        result.append(Person(i["name"], i["age"]))
+    for i in people:
+        person = Person.people.get(i["name"])
+        if "wife" in i and i["wife"]:
+            wife_person = Person.people.get(i["wife"])
+            person.wife = wife_person
+            wife_person.husband = person
+        elif "husband" in i and i["husband"]:
+            husband_person = Person.people.get(i["husband"])
+            person.husband = husband_person
+            husband_person.wife = person
```

Comment:
Similar to the previous comment, these lines assume that `husband_person` and `person` have `husband` and `wife` attributes, respectively. Ensure these attributes are defined or handle cases where they might not exist.